.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 bower_components
 dist
 config.json
+npm-debug.log**
```

Comment:
you should also add `*.zip` to prevent pushing of the generated zip file

Reply:
Added @rhenshaw56